src/__tests__/Profile.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import Profile from '../Components/MyProfilePage';
+import store from '../redux/store';
+
+test('Check if MyProfile component is rendered correctly', () => {
```

Comment:
- [ ] Excellent work implementing some tests for the capstone! It would be great to add a couple of more tests to it. Kindly add tests for other components from your project to make it more stable and scalable! 💯 💯